operations/mimir/config.libsonnet

```diff
@@ -74,6 +74,10 @@
 
     jaeger_agent_host: null,
 
+    // Allow to configure the alertmanager disk.
+    alertmanager_data_disk_size: '100Gi',
+    alertmanager_data_disk_class: 'fast',
```

Comment:
Does it have to be fast? I feel it should be enough with a `standard` class.

Reply:
`standard` would probably be fine as well, I will updated that